third_party/cpgfunctionEP-0.3.2/include/cpgfunction/heat_transfer.h

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
Functions to compute the finite line source solutions. Ie. segment-to-segment response matrix.